tests/multi_ptr/multi_ptr_arithmetic_op.h

```diff
@@ -0,0 +1,417 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides code for multi_ptr arithmetic operators
+//
+*******************************************************************************/
+
+#ifndef __SYCLCTS_TESTS_MULTI_PTR_ARITHMETIC_OP_H
+#define __SYCLCTS_TESTS_MULTI_PTR_ARITHMETIC_OP_H
+
+#include "../common/common.h"
+
+#include "../common/section_name_builder.h"
+#include "multi_ptr_common.h"
+
+namespace multi_ptr_prefetch_member {
```

Comment:
prefetch?

Reply:
yes, of course, fixed in this commit: https://github.com/KhronosGroup/SYCL-CTS/pull/353/commits/c05d4cdb323420a26af586853442ec0fea0e7ee4